Tools/autotest/swarminit.txt

```diff
@@ -0,0 +1,6 @@
+0=0,0,0,0
```

Comment:
it would be good to have some # prefix comments in this file explaining the fields